assets/scss/_changelog.scss

```diff
@@ -0,0 +1,9 @@
+.changelog-fix {

```

Comment:
The should be a style for an "API" type.

Reply:
All styles were removed as they are not used.